platform/darwin/src/MGLCluster.h

```diff
@@ -0,0 +1,54 @@
+#import <Foundation/Foundation.h>
+#import "MGLFoundation.h"
+
+@protocol MGLFeature;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ An `NSUInteger` constant used to indicate an invalid cluster identifier.
+ This indicates a missing cluster feature.
+ */
+FOUNDATION_EXTERN MGL_EXPORT const NSUInteger MGLClusterIdentifierInvalid;
+
+/**
+ A protocol that (private) feature subclasses (i.e. those already conforming to
+ the `MGLFeature` protocol) conform to if they are represent clusters.
+ 
+ Currently only subclasses of `MGLPointFeature` support `MGLCluster`.
```

Comment:
This only says that we aren’t defining additional classes that conform to the MGLCluster protocol. However, it doesn’t dissuade developers from defining their own classes that conform to the protocol and creating instances of those classes, which is technically possible and might be tempting under some circumstances.

I’m curious why we wound up making this a protocol instead of a subclass of MGLPointFeature. Is it expected that clusters of non-point features could be inspected in the future?